tools/reviewer/web/src/app/dashboard/diffs/diffs.component.ts

```diff
@@ -53,75 +54,81 @@ export class DiffsComponent implements OnInit {
 
     if (urlEmail) {
       // Show the page from a view of the user from url.
-      this.getReviews(urlEmail);
+      this.categorizeDiffs(urlEmail);
     } else {
       // Show the page from current login view.
-      this.getReviews(this.authService.userEmail);
+      this.categorizeDiffs(this.authService.userEmail);
     }
   }
 
-  // Get diff/reviews from Database
-  getReviews(userEmail: string) {
-    this.firebaseService.getDiffs().subscribe(
-      diffs => {
-        // Diffs are categorized in 4 different lists
-        this.diffGroups[DiffGroups.NeedAttention] = [];
-        this.diffGroups[DiffGroups.Incoming] = [];
-        this.diffGroups[DiffGroups.Outgoing] = [];
-        this.diffGroups[DiffGroups.CC] = [];
-        this.diffGroups[DiffGroups.Draft] = [];
-        this.diffGroups[DiffGroups.Pending] = [];
-        this.diffGroups[DiffGroups.Submitted] = [];
+  // Categorize diffs in specific groups
+  categorizeDiffs(userEmail: string) {
+    this.firebaseService.getDiffs().subscribe(diffs => {
+      this.diffGroups[DiffGroups.NeedAttention] = [];
+      this.diffGroups[DiffGroups.Incoming] = [];
+      this.diffGroups[DiffGroups.Outgoing] = [];
+      this.diffGroups[DiffGroups.CC] = [];
+      this.diffGroups[DiffGroups.Draft] = [];
+      this.diffGroups[DiffGroups.Pending] = [];
+      this.diffGroups[DiffGroups.Submitted] = [];
 
-        // Iterate over each object in res
-        // and create Diff from proto and categorize
-        // into a specific list.
-        for (const diff of diffs) {
-          // needAttentionOfList is a list of all reviewers,
-          // where attention is requested
-          const needAttentionOfList: string[] = diff.getReviewerList()
-            .filter(reviewer => reviewer.getNeedsAttention())
-            .map(reviewer => reviewer.getEmail());
-
-          if (diff.getAuthor().getEmail() === userEmail) {
-            // Current user is an author of the diff
-            this.diffGroups[DiffGroups.Incoming].push(diff);
+      for (const diff of diffs) {
+        if (diff.getAuthor().getEmail() === userEmail) {
+          // Current user is an author of the diff
+          switch (diff.getStatus()) {
+            case Diff.Status.UNDER_REVIEW:
+            case Diff.Status.NEEDS_MORE_WORK:
+            case Diff.Status.ACCEPTED:
+            case Diff.Status.SUBMITTING:
+            case Diff.Status.REVERTING:
+              // Outgoing Review
+              this.diffGroups[DiffGroups.Outgoing].push(diff);
+              break;
+            case Diff.Status.SUBMITTED:
```

Comment:
Let's add REVERTED here, together with SUBMITTED.
Reason: REVERTED is a specific case of SUBMITTED.